### ISSUE TITLE
html.js parser may need a fix

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#2606]
see: https://github.com/Jermolene/TiddlyWiki5/issues/2604#issuecomment-253189016

to reproduce: 

```
<bob@example.com>some text</bob@example.com>
```

doesn't work, but the following does:

```
<bob@example.com>some text</bob>
```

The browser produces a DOM like this: `<bob>some text</bob>`  which is ignoring `@xxxx`

**Comment by pmario** [comment]
While we are at this: `textPrimitives` are set at several different positions (use global search) in the source code. and eg: "anyLetter" may have several different definitions. ... IMO that should be fixed too.

**Comment by Jermolene** [comment]
> The browser produces a DOM like this: <bob>some text</bob> which is ignoring @xxxx

If you use the "parse tree" view for the preview pane, you'll see the `@xxx` portion is actually being wrongly parsed as an attribute!

Anyhow, I think the fix for #2604 also fixes this issue, what do you think?

> While we are at this: textPrimitives are set at several different positions (use global search) in the source code. and eg: "anyLetter" may have several different definitions. ... IMO that should be fixed too

I agree, but it's a separate issue from the OP here.

**Comment by pmario** [comment]
seems to be fixed with #2604
